writing/paper.md

```diff
@@ -2,84 +2,243 @@
+
+Primary Resources:
+1. The paper "Natural Language Processing Model for Automatic Analysis of Cybersecurity-Related Documents" by Tiberiu-Marian Georgescu describes the development of an NLP model for automatic analysis of cybersecurity-related documents. (1) 
+
+2. "Cyber Security Vulnerability Detection Using Natural Language Processing" This paper expands a scheme targeting spreadsheet exposure discovery as an NLP question. (2)
+
+3. The SAMHSA Publications and Digital Products primer, "Communicating in a Crisis," offers advice on how to communicate effectively and make decisions. The primer emphasizes the value of concise, accurate, and clear communication during a crisis and offers advice on how to communicate effectively, such as by sending out regular updates and using simple messages. The primer offers insights into the significance of effective communication during a crisis, which may be pertinent to the development of NLP models for detecting and responding to cybersecurity threats. (3)
+
 
-TODO: This section includes a broad and detailed review of relevant existing work. The literature review should provide background and context for the reseach project work. The subsections may be organized in whatever manner seems best suited to the material-- chronological, or by topic, or according to some other criteria (e.g., primary versus secondary resources).
+Secondary Resources:
+1. A research paper examining the available literature and open data sources related to cybersecurity and cyber risk is titled "Cyber risk and cybersecurity: a systematic review of data availability." The paper, which offers insights into the accessibility of cybersecurity and cyber risk data, is accessible on the NCBI website. About 150 articles, business and government reports, academic papers, and historical cost data or derived cost estimates for losses make up the paper. As it offers insights into the available literature and data sources related to cybersecurity and cyber risk. (4)
+
+
+2. The IEEE Xplore paper, "Review of NLP-based Systems in Digital Forensics and Cybersecurity," reviews the literature on these systems in cybersecurity and digital forensics, outlining their functions, uses, difficulties, and potential future directions. It sheds light on how NLP-based systems are used in cybersecurity and digital forensics. The paper identifies challenges and potential research directions in this field as well as the potential of NLP-based systems for detecting and responding to cybersecurity threats. (5)
+
+
+![Spam-detection-block-diagram](https://github.com/LS102-summer2023/project-mishikakansal/assets/136472000/ccad79c2-a95c-4d4b-91e8-2c966861e295)        (23)
 
-If ethical issues are central to this work, you should also address historical and contemporary issues or efforts made to address them.
 
 ### Prototype
```

Comment:
What is the goal of this prototype? How does it address your stated project goals?